src/assets/style/breakPoints.scss

```diff
@@ -0,0 +1,2 @@
+$screen-sm: 767px;
+$screen-md: 1023px;
```

Comment:
Is there any reason to set px like this? typically tailwind, bootstap follows 768 for 1024 px